apps/src/javalab/Javalab.js

```diff
@@ -90,6 +91,8 @@ Javalab.prototype.init = function(config) {
     visualization = <DefaultVisualization />;
   }
 
+  const handleVersionHistory = this.studioApp_.getVersionHistoryHandler(config);
```

Comment:
Just a quick reminder to move this up to where the other handlers are declared (i.e. by `onInputMessage` above)

Reply:
🙌